.changes/1078.json

```
@@ -0,0 +1,5 @@
+{
+    "type": "fixed",
+    "description": "support custom images in packages with the `.*\\d*-$` package name pattern.",
```

this is a bit more general than I think is needed, maybe

```suggestion
    "description": "Fix custom image names for images with a trailing '-' character.\ncustom images in packages with the `.*\\d*-$` package name pattern are now supported.",

---

also maybe mention that digits are now taken into account too

---

Technically digits were accepted before, they were just ignored. If they were used internally, such as `foo123bar`, it would produce a custom image tag of `foobar`, and at the end for `foo123`, it would be `foo`. So the custom image name has changed, but only packages with the `.*\\d*-$` name pattern were affected.

---

The correct regex would for what's now accepted but what would previously be considered invalid is `^.*([^A-Za-z_]*-)|(-[^A-Za-z_]*)$`. I can update the changelog entry with that.

---

Is the changelog entry better? Feel free to merge when ready.